docs/transparent-validation/protectli/laboratory-assembly-guide.md

```diff
@@ -3,7 +3,7 @@
 ## Introduction
 
 This document describes platform-specific details for assembling Protectli
-VP2410, VP2420, VP4630/VP4650/VP4670, V1210/V1410/V1610 testing stands.
+VP2410, VP2420, VP4630/VP4650/VP4670, V1210/V1410/V1610, VP66xx testing stands.
```

Comment:
Let's use XX in others, as we do for VP66

Reply:
Changed in: 5084c58dada6befae99d63f2a81cb35ccb4a4962